CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/providers/TransactionTypeNodeProvider.java

```diff
@@ -291,16 +395,20 @@ public boolean isLeaf() {
                     // Any vertextype that points to itself is in the root layer.
                     for (final SchemaTransactionType tt : transactionTypes) {
                         if (tt.getSuperType() == tt) {
+                            if (isFilterMatchCurrentNode(tt) || filterText.getText().toLowerCase().isEmpty()) {
```

Comment:
Can probably omit `.toLowerCase()` if you are just checking if it's empty

Reply:
Ah yes. Fixed it.